examples/language-modeling/README.md

```diff
@@ -746,7 +746,7 @@ python3 ../text-generation/run_generation.py \
     --use_kv_cache \
     --batch_size 1 \
     --use_hpu_graphs \
-    --ignore_eos \
+    --no-ignore_eos \
```

Comment:
Why change `--ignore_eos` to `--no-ignore_eos` only for this cmd?

Reply:
The cmd is shown in Prompt/Prefix/P-tuning section, which uses https://huggingface.co/datasets/ought/raft/viewer/twitter_complaints for finetune, whose label is expected as "complaint" or "no complaint", as a result, the generation output is expected as "complaint" or "no complaint" using the finetuned peft model as input. if not set --no-ignore_eos, extra work will be output.